ledger/controllers/wallet.js

```diff
@@ -375,6 +350,8 @@ const write = function (runtime, apiVersion) {
       }
     }
 
+    result = await runtime.wallet.redeem(wallet, txn, signedTx, request)
```

Comment:
we can delete this as well - that way we're only using transferring funds via `submitTx` below